src/Traits/HasSharedLogic.php

```diff
@@ -32,6 +35,41 @@ public function to(int|string $chatId): self
         return $this;
     }
 
+    /**
+     * sets reply markup for payload
+     *
+     *
+     * @return static
+     *
+     * @throws \JsonException
+     */
+    public function reply(array $markup): self
```

Comment:
I think we should rename this to `replyMarkup` or `keyboardMarkup` to avoid confusion.

Reply:
ok